src/coreclr/jit/assertionprop.cpp

```diff
@@ -694,7 +694,7 @@ void Compiler::optAssertionInit(bool isLocalProp)
         // more than 64 assertions.
         // Note this tracks at most only 256 assertions.
         //
-        static const AssertionIndex countFunc[] = {64, 128, 256, 64};
+        static const AssertionIndex countFunc[] = {64, 128, 256, 128, 64};
```

Comment:
Allowed a bit more assertions for large methods - it was done mainly to fix asm regressions introduced by new asserts where old ones were thrown away due to this limit.